spec/models/book_spec.rb

```diff
@@ -6,6 +6,11 @@
     it { should validate_presence_of(:author) }
 
     subject { FactoryGirl.build(:book) }
-    it { should validate_uniqueness_of(:title).case_insensitive}
+    it { should validate_uniqueness_of(:title).case_insensitive }
+  end
+
+  describe 'associations' do
+    it { should have_many(:checkouts) }
```

Comment:
I usually wouldn't bother with specs that assert that an association exists. If the association is useful, it'll likely be covered by some feature spec, as it is here. That said, some people like these and if that's the case I don't feel strongly that you should remove them. Just thought I'd mention it.